bundler/lib/dependabot/bundler/helpers.rb

```diff
@@ -30,9 +30,9 @@ def self.detected_bundler_version(lockfile)
         return "unknown" unless lockfile
 
         if (matches = lockfile.content.match(BUNDLER_MAJOR_VERSION_REGEX))
-          matches[:version]
+          matches[:version].to_i
         else
-          "1"
+          "unknown"
```

Comment:
Maybe to differentiate from having no lockfile?

```suggestion
          "unspecified"
```